### PR TITLE
PowerScale Obs doc update for CSM Observability 1.3.0

### PR DESCRIPTION
**Pull request by baoy1**
# Description
Add CSM observability for PowerScale to csm-docs, and update docs for online/offline install and upgrade.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/379|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by taohe1012 — APPROVED** [review]

**Review by taohe1012 — APPROVED** [review]

File: content/docs/observability/_index.md [hunk 1]
```diff
@@ -14,13 +14,14 @@ Description: >
 
 Metrics data is collected and pushed to the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector), so it can be processed, and exported in a format consumable by Prometheus. SSL certificates for TLS between nodes are handled by [cert-manager](https://github.com/jetstack/cert-manager).
 
-CSM for Observability is composed of several services, each living in its own GitHub repository, that can be installed following one of the three deployments we support [here](deployment). Contributions can be made to this repository or any of the CSM for Observability repositories listed below. 
+CSM for Observability is composed of several services, each living in its own GitHub repository, that can be installed following one of the four deployments we support [here](deployment). Contributions can be made to this repository or any of the CSM for Observability repositories listed below.
```
Comment (rsedlock1958) [inline content/docs/observability/_index.md:17]: Delete "below"

File: content/docs/observability/deployment/helm.md [hunk 2]
```diff
@@ -38,19 +39,38 @@ The Container Storage Modules (CSM) for Observability Helm chart bootstraps an O
...
 
         `kubectl get secret powerstore-config -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -`
 
+    ### PowerScale
+
+    1. Copy the config Secret from the CSI PowerScale namespace into the CSM for Observability namespace:
+
+       `kubectl get secret isilon-creds -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -` 
+
+    If [CSM for Authorization is enabled](../../../authorization/deployment/#configuring-a-dell-csi-driver-with-csm-for-authorization) for CSI PowerScale, perform the following steps:
```
Comment (rsedlock1958) [inline content/docs/observability/deployment/helm.md:54]: Change "the following" to "these"

File: content/docs/observability/upgrade/_index.md [hunk 3]
```diff
@@ -80,3 +80,42 @@ CSM for Observability online installer upgrade can be used if the initial deploy
     |
     |- Waiting for pods in namespace karavi to be ready                 Success
     ``` 
+
+## Offline Installer Upgrade
+
+Assuming that you have already installed the Karavi Observability Helm Chart by offline installer and meet its installation requirement.
+The following instructions can be followed when a Helm chart was installed and will be upgraded in an environment that does not have an internet connection and will be unable to download the Helm chart and related Docker images.
```
Comment (rsedlock1958) [inline content/docs/observability/upgrade/_index.md:87]: Change "The following" to "These"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. Minor edits required.

File: content/docs/observability/_index.md (hunk 1 above)
Comment (baoy1) [reply content/docs/observability/_index.md:17]:
```
{{<table "table table-striped table-bordered table-sm">}}
```
Probably github cannot render the HTML tag properly. Same for other branches,  e.g. [release-1.4-updated](https://github.com/dell/csm-docs/blob/release-1.4-updated/content/docs/observability/_index.md).
After publishing the doc using hugo, the HTML is correct, see http://10.225.6.31:1313/csm-docs/docs/observability/

File: content/docs/observability/deployment/helm.md (hunk 2 above)
Comment (baoy1) [reply content/docs/observability/deployment/helm.md]: Address.

File: content/docs/observability/upgrade/_index.md (hunk 3 above)
Comment (baoy1) [reply content/docs/observability/upgrade/_index.md]: Address.

**Comment by baoy1** [comment]
> I have reviewed this. Minor edits required.

All addressed. Thank you!

**Review by forrestxia — APPROVED** [review]

**Review by taohe1012 — APPROVED** [review]

**Review by P-Cao — APPROVED** [review]

**Review by YianZong — APPROVED** [review]

**Comment by baoy1** [comment]
Hi @rsedlock1958 , @gallacher, @mjsdell , @prablr79, @rajendraindukuri, @
shanmydell, any more comments prior to your approvals?

**Review by shanmydell — APPROVED** [review]

**Pull request merged**